geo-types/src/macros.rs

```diff
@@ -217,6 +245,19 @@ macro_rules! line_string {
 #[macro_export]
 macro_rules! polygon {
     () => { $crate::Polygon::new($crate::line_string![], vec![]) };
+    ($($x:literal $y:literal),+) => {
+        polygon!($(coord! { x: $x, y: $y }),+)
```

Comment:
I think you can use `$crate::coord` here to avoid requiring the user to import it.

Reply:
ok, thanks for help!

Reply:
Fixed it.